_docs/.custom_wordlist.txt

```diff
@@ -2,6 +2,7 @@
 autocomplete
 autocompletions
 args
+CNI
```

Comment:
I'm not convinced that CNI is a well-known enough term that we're ok using it as an abbreviation. If we spell it out, we also avoid having to have it here.